api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +50,17 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String accessLocationUuid = Context.getAdministrationService().getGlobalProperty("locationbasedaccess.locationAttributeUuid");
```

Comment:
Can you use a constant for the global property name?

Reply:
The variable name "accessLocationUuid" is not very meaningful.

Reply:
Sure, I will change it